github-templates/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+### Summary
+
+### Expected behaviour
+
+### Actual behaviour
+
+### Steps to reproduce
```

Comment:
What if the ISSUE isn't a bug report, but a story to cover an enhancement or new functionality? Or are ISSUEs only ever bug reports?